packages/shorebird_cli/test/src/commands/release/release_ios_command_test.dart

```diff
@@ -635,9 +635,7 @@ error: exportArchive: No signing certificate "iOS Distribution" found
       );
 
       expect(exitCode, equals(ExitCode.software.code));
-      verify(
-        () => logger.err(any(that: contains('No directory found'))),
```

Comment:
Where was this No directory found previously coming from?

Reply:
Ah nvm it was from the remaining error when an ipa file was not found.

Reply:
https://github.com/shorebirdtech/shorebird/blob/bo/unsigned-ipa/packages/shorebird_cli/lib/src/shorebird_artifact_mixin.dart#L127